module/support/logging/FileLogHandler/src/FileLogHandler.cpp

```diff
@@ -16,34 +18,33 @@ namespace module::support::logging {
 
     FileLogHandler::FileLogHandler(std::unique_ptr<NUClear::Environment> environment)
         : Reactor(std::move(environment))
-        , mutex()
-        , logFileName("/home/nubots/log")
-        , logFile(logFileName, std::ios_base::out | std::ios_base::app | std::ios_base::ate) {
+        , log_file_name("/home/nubots/log")
+        , log_file(log_file_name, std::ios_base::out | std::ios_base::app | std::ios_base::ate) {
 
         on<Configuration>("FileLogHandler.yaml").then([this](const Configuration& config) {
             // Use configuration here from file FileLogHandler.yaml
-            logFileName = config["log_file"].as<std::string>();
+            log_file_name = std::filesystem::path(config["log_file"].as<std::string>());
+
+            max_size = config["max_size"].as<long>();
```

Comment:
You will need to include `yaml_expression.hpp` for this
```suggestion
            max_size = config["max_size"].as<Expression>();
```